### ISSUE TITLE
Give new discord server link the old one(s) are/is dead

### ISSUE THREAD

**Issue by YUG38** [NightProxy/Space#6]

**Comment by nonsnse** [comment]
This will be fixed soon. Not exactly an issue but good to know.

**Comment by Mepoify** [comment]
Yeah I'm having the same problem so I just cloned and deployed on my own Vercel account for a personal link.
[spaceclone.vercel.app]()

**Comment by xnoctra** [comment]
Alright that's fine

________________________________
